games-util/lutris/lutris-9999.ebuild

```diff
@@ -27,9 +27,12 @@ RDEPEND="
 	dev-python/pygobject:3[${PYTHON_USEDEP}]
 	dev-python/python-evdev[${PYTHON_USEDEP}]
 	dev-python/pyyaml[${PYTHON_USEDEP}]
+	gnome-base/gnome-desktop
 	net-libs/libsoup
+	net-libs/webkit-gtk
```

Comment:
net-libs/webkit-gtk:4[introspection]

Reply:
still appears to be missing slot (and probably slot operator as well?)